docs/snippets/common/msw-addon-initialize.js.mdx

```diff
@@ -0,0 +1,16 @@
+```js
+// .storybook/preview.js
+import { initialize, mswLoader } from 'msw-storybook-addon';
+
+/*
+ * Initializes MSW
+ * See https://github.com/mswjs/msw-storybook-addon#configuring-msw
+ * to learn how to customize it
+ */
+initialize();
+
+export default {
```

Comment:
Not sure if this is used across the JS docs, but you could add types like so:

/**
 * @type {import('@storybook/react').Preview}
 */

Reply:
That's a great idea, but should be a separate effort.